### PR TITLE
Database Write Function

### PR DESCRIPTION
**Pull request by Ajevan**
Writes student data to Database:

Screenshot below of two user test:
Screen Shot 2021-11-26 at 2 54 35 PM

### CODE REVIEW

File: pages/teachers/djacobs/giza.tsx [hunk 1]
```diff
@@ -74,6 +81,50 @@ export default function djacobs(props) {
...
   };
 
+  const [insertGizaStudentData, insertGizaStudentDataMutation] = useMutation(
+    ADD_GIZA_DATA,
+    {
+      onCompleted: (data) => {
+        const ID = data.insert_giza_student_grades.returning[0].id;
+        const studentName =
+          data.insert_giza_student_grades.returning[0].student_name;
+        console.log(ID);
```
Comment (vithushan19) [inline pages/teachers/djacobs/giza.tsx:91]: Remove logs

File: pages/teachers/djacobs/giza.tsx [hunk 2]
```diff
@@ -74,6 +81,50 @@ export default function djacobs(props) {
     console.log(guess);
   };
 
+  const [insertGizaStudentData, insertGizaStudentDataMutation] = useMutation(
+    ADD_GIZA_DATA,
+    {
+      onCompleted: (data) => {
```
Comment (vithushan19) [inline pages/teachers/djacobs/giza.tsx:87]: You're not even doing anything here besides logging. Just remove the function

File: pages/teachers/djacobs/giza.tsx [hunk 3]
```diff
@@ -74,6 +81,50 @@ export default function djacobs(props) {
...
+
+  const insertGizaStudentDataFunction = (
+    studentName: string,
+    groupName: string,
+    guesses: GuessData[],
+    guessHistory: GuessResponse[],
+    startTime: number,
+    endTime: number
+  ) => {
+    //console.log(studentName);
```
Comment (vithushan19) [inline pages/teachers/djacobs/giza.tsx:105]: Remove

File: pages/teachers/djacobs/giza.tsx [hunk 4]
```diff
@@ -126,7 +177,10 @@ export default function djacobs(props) {
   };
 
   const isWrong = (check: Boolean, guess: GuessData) => {
-    guessHistory.set("Question" + questionCounter + "." + guessCounter, guess);
+    //"Question" + questionCounter + "." + guessCounter, guess
```
Comment (vithushan19) [inline pages/teachers/djacobs/giza.tsx:180]: Remove

**Review by vithushan19 — APPROVED** [review]

File: pages/teachers/djacobs/giza.tsx (hunk 1 above)
Comment (Ajevan) [reply pages/teachers/djacobs/giza.tsx]: Resolved in latest commit

File: pages/teachers/djacobs/giza.tsx (hunk 2 above)
Comment (Ajevan) [reply pages/teachers/djacobs/giza.tsx]: Resolved in latest commit

File: pages/teachers/djacobs/giza.tsx (hunk 3 above)
Comment (Ajevan) [reply pages/teachers/djacobs/giza.tsx]: Resolved in latest commit

File: pages/teachers/djacobs/giza.tsx (hunk 4 above)
Comment (Ajevan) [reply pages/teachers/djacobs/giza.tsx]: Resolved in latest commit

**Pull request merged**